### ISSUE TITLE
Mistake in KL divergence formula

### ISSUE THREAD

**Issue by ghost** [pat-coady/trpo#25]
Hi,

There is a small mistake in the policy.py file [when you calculate the kl divergence between two multivariate normal distributions](https://github.com/pat-coady/trpo/blob/c5050f4b51dc3d6cf4e3e13c3a034fbb9f385937/src/policy.py#L127) :

`         self.kl = 0.5 * tf.reduce_mean(log_det_cov_new - log_det_cov_old + tr_old_new +
                                       tf.reduce_sum(tf.square(self.means - self.old_means_ph) /
                                                     tf.exp(self.log_vars), axis=1) -
                                       self.act_dim)`

The ratio of the covariances i.e. `tr_old_new` should be squared in the KL divergence i.e. `tr_old_new` just needs to be replaced with `tr_old_new**2`.
